### PR TITLE
Fix broken test.

### PR DESCRIPTION
**Pull request by sallner**
And make some things nicer.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,15 +1,13 @@
 language: python
+sudo: false
```
Comment (dataflake) [inline .travis.yml]: I don't think this is required

Comment (mgedmin) [reply .travis.yml]: Yeah, check out the [Config tab](https://travis-ci.org/github/zopefoundation/zc.catalog/builds/686485939/config) on the Travis build for this PR:

> <code>root</code>: deprecated key <code>sudo</code> (The key `sudo` has no effect anymore.)

Comment (stevepiercy) [reply .travis.yml]: It is deprecated. Check the [Travis build config](https://travis-ci.org/github/zopefoundation/zc.catalog/builds/686485939/config) and click "View config" to see the deprecation warning.

**Review by mgedmin — APPROVED** [review]
LGTM.

File: .travis.yml (hunk 1 above)
Comment (sallner) [reply .travis.yml]: I changed it.

**Review by dataflake — APPROVED** [review]

**Pull request merged**